include/sound/sof.h

```diff
@@ -106,6 +106,7 @@ struct snd_sof_pdata {
 	const char *fw_filename;
 	const char *tplg_filename_prefix;
 	const char *tplg_filename;
+	bool disable_function_topology;
```

Comment:
@bardliao can you modify the commit message to say "Add a kernel module parameter to override loading function topologies when set"?

Reply:
> @bardliao can you modify the commit message to say "Add a kernel module parameter to override loading function topologies when set"?

@ranj063 But the `disable_function_topology` module parameter doesn't use this flag. From @ujfalusi's point of view, the `disable_function_topology` flag and the  `disable_function_topology` module parameter are 2 different parameters.
https://github.com/thesofproject/linux/pull/5373#issuecomment-2747307500

Do you prefer we set the `disable_function_topology` flag if the  `disable_function_topology` module parameter is set?
I am all ears.

Reply:
Ahh I see. Sorry i got confused there.